src/components/scene/vr-mode-ui.js

```diff
@@ -71,6 +45,8 @@ module.exports.Component = registerComponent('vr-mode-ui', {
 
     this.orientationModalEl = createOrientationModal(this.exitVR);
     this.el.appendChild(this.orientationModalEl);
+
+    this.attachEventListeners();
```

Comment:
I would move the creation (without appending) of the ```Els``` into ```init``` so that we can get the event listeners attached in ```init```. The danger of this is that if a property changes, then we will end up registering duplicate listeners.